lib/runtime-core/src/backing.rs

```diff
@@ -449,10 +449,17 @@ fn import_functions(
                 });
             }
             None => {
-                link_errors.push(LinkError::ImportNotFound {
-                    namespace: namespace.to_string(),
-                    name: name.to_string(),
-                });
+                if imports.allow_missing_functions {
+                    functions.push(vm::ImportedFunc {
+                        func: ::std::ptr::null(),
+                        vmctx: ::std::ptr::null_mut(),
```

Comment:
It would be nice if we could mock missing imports again (which is a separate feature though). We used to mock imports and just print a message like "mock import `func_test` called".

Reply:
With the currently implementation the program receives a SIGSEGV when a null imported function is called, which I think is fine for now.

The import mocking feature can be added in a separate PR.